rspec-github.gemspec

```diff
@@ -23,6 +23,6 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'rspec-core', '~> 3.0'
   spec.add_development_dependency 'rspec', '~> 3.0'
-  spec.add_development_dependency 'rubocop', '~> 1.12.0'
-  spec.add_development_dependency 'rubocop-ast', '~> 1.4.0'
+  spec.add_development_dependency 'rubocop', '~> 1.12'
+  spec.metadata['rubygems_mfa_required'] = 'true'
```

Comment:
This will change the publishing, but I see it is sent on another one of your projects https://github.com/Drieam/sql_attributes/blob/d8536c0ae576c02e7ca3576a192048853e6ea5d5/sql_attributes.gemspec#L37